keycodes.go

```diff
@@ -0,0 +1,261 @@
+package main
+
+import (
+	"strconv"
+	"strings"
+)
+
+func formatKeycodes(keycode string) string {
+	kc := map[string]int{
```

Comment:
This is probably a map we only want initialized once, so it doesn't have to be allocated and freed for every call of this function. I'm fine with using global vars in such cases.